### ISSUE TITLE
call of duty warzone

### ISSUE THREAD

**Issue by appleface80** [ogurets/popcnt_emulator#6]
I have a Xeon e5450.
how would I use the emulator to be able to play warzone.

**Comment by A-Flux** [comment]
Same problem with x5450.
I would need the dll´s for using it!

**Comment by ghost** [comment]
It is in [https://github.com/ogurets/popcnt_emulator/releases](https://github.com/ogurets/popcnt_emulator/releases).

**Comment by appleface80** [comment]
I don't understand how I'm supposed to yes that to play warzone. could you please explain what to do.

**Comment by A-Flux** [comment]
Thank you.

But, if i start it with
pin.exe -follow-execv -t64 instruction_hook64.dll -t instruction_hook32.dll -- "\Call of Duty Modern Warfare\ModernWarfare.exe"
the Executable ModernWarfare.exe stays forever in background with stable 12% CPU.

**Comment by ghost** [comment]
Yes, the same here. I just upgraded machine to old xeon x3430 which has support for popcnt.

**Comment by mirh** [comment]
That's because the exe needs like two fucktons of command line parameters passed from the battle.net client to work. 
You may want to try to make it passthrough them to pin.exe by tinkering with `%PROGRAMDATA%\Battle.net\Agent\data\cache\cb\78\cb78a3a67134163cd6016efdb8d9cf59`

**Comment by AyushKumar-Codes** [comment]
What do it mean i can't understand.. I am facing same problem in Marvel Avengers. It only runs in background processes. Please help me..

**Comment by WaveWavovich** [comment]
> That's because the exe needs like two fucktons of command line parameters passed from the battle.net client to work.
> You may want to try to make it passthrough them to pin.exe by tinkering with `%PROGRAMDATA%\Battle.net\Agent\data\cache\cb\78\cb78a3a67134163cd6016efdb8d9cf59`

sorry to bother, but can you, please, explain, what does tinkering means? also, this cache file's name and the folder where 
it is changed its name, and yeah, how do I "extract" these command line parameters from just a cache file and make it pass through to pin.exe? 

thanks in advance

**Comment by mirh** [comment]
New hash-ID should be [here](https://blizztrack.com/v/odin/versions/389315). 
Tinkering means editing. 
I can't really help much more.

**Comment by ghost** [comment]
@mirh  @kukulo2011 @NegativG @appleface80 @A-Flux hello, did you solve your problem ?
I'm facing the same issue, I can't load COD Warzone on my Xeon E5450 because of lack of SSE4.2 instructions.

If you have any ideas to solve this problem it would be great !

**Comment by AyushKumar-Codes** [comment]
Nothing will help you .. You have to upgrade your Processor to intel I3 to
i7. Then only you could play this game.. These emulator won't works
